nltk/tokenize/stanford_segmenter.py

```diff
@@ -64,15 +63,9 @@ def __init__(self,
                 env_vars=('STANFORD_SEGMENTER',),
                 searchpath=(), url=_stanford_url,
                 verbose=verbose)
-        slf4j = find_jar(
```

Comment:
Did you try my proposal? Something like:
```python
StanfordSegmenter(path_to_slf4j=None)
```

you have set `path_to_slf4j` to force exclusion of the `slf4j`. This is explicit and compatible with the current behavior.